### ISSUE TITLE
Question about training result.

### ISSUE THREAD

**Issue by linjing7** [mks0601/Hand4Whole_RELEASE#28]
Hi, thank you very much for your excellent.

I trained the body-only model by running `python train.py --gpu 0-3 --parts body` and then tested it on the PW3D dataset by running `python test.py --gpu 0-3 --parts body --test_epoch 6`.  The testing result is `MPJPE: 90.58 mm, PA MPJPE: 56.90 mm, MPVPE: 108.59 mm, PA MPVPE: 76.12 mm`, which is about 5mm worse than the result tested with the provided pretrianed model. Could you please tell me about the reason? Is it caused by deviation? Or does the provided model have been finetuned in the PW3D dataset?

**Comment by mks0601** [comment]
The provided model is not fine-tuned on 3DPW. The primary metric is PA MPJPE, and it seems similar? Which metric are you interested in?

**Comment by linjing7** [comment]
Thanks for your timely reply. It seems that MPVPE and PA-MPVPE are poor. Is it normal?  BTW,  I do not train body-only Pose2Pose from the pre-trained 2D human pose estimation network by running `python train.py --gpu 0-3 --parts body --continue`.  Will it influence the final result?

**Comment by mks0601** [comment]
Yeah it could affect. Could you train again with that option?

**Comment by linjing7** [comment]
Yes, I'm willing to do it. But there is something wrong when I train with that option? Need I rename the pretrained model from "body_pose_resnet_50_256x192.pth.tar" to something like "snapshot_6.pth.tar"? 
![image](https://user-images.githubusercontent.com/70081899/185025137-4d40b9c0-c25c-49d6-9ea4-de7c384705c6.png)

**Comment by mks0601** [comment]
Sorry you should rename it to snapshot_0.pth.tar before running the training command. Will clarify this in README

**Comment by linjing7** [comment]
Okay, thank you very much.  There still exists something wrong when I rename it to  snapshot_0.pth.tar. It is caused by the reason that "snapshot_0.pth.tar" does not contain the epoch information. 
![image](https://user-images.githubusercontent.com/70081899/185028681-9dca255b-9b1f-4998-8fe5-38b6eb691796.png)

And I fix it by replacing `start_epoch = ckpt['epoch'] + 1
model.load_state_dict(ckpt['network'], strict=False)` in common/base.py with `start_epoch = cur_epoch
        model.load_state_dict(ckpt, strict=False)` .   And now it can be trained successfully.  

Thank you very much for your patient reply and I hope this information can help you :)

**Comment by mks0601** [comment]
Sorry there was a mistake :(
1. Place `body_pose_resnet_50_256x192.pth.tar` to `tool` folder and run `python convert_simple_to_pose2pose.py `.
2. Move generated `snapshot_0.pth.tar` to `output/model_dump`.
3. Run the training commands with `--continue` option.

**Comment by linjing7** [comment]
Okay, it works when I follow your instruction. Thank you very much!

**Comment by linjing7** [comment]
Hi, I've retrained with `--continue` option and can obtain the result the same as the paper. Could you please tell me how the 2D human pose estimation network "body_pose_resnet_50_256x192.pth.tar" is pretrained?

**Comment by mks0601** [comment]
As written in the paper, it is from SimpleBaseline (ECCV 2018).
It is just renamed one, downloaded from https://github.com/microsoft/human-pose-estimation.pytorch#quick-start.
You can download pose_resnet_50_256x192.pth.tar

**Comment by linjing7** [comment]
Okay, thank you very much!
